src/components/VR/vr.js

```diff
@@ -4,7 +4,7 @@ const VR = () => {
 
     return (
         <div className='pt-10' style={{ textAlign: 'center', marginTop: '50px' }}>
-            <h1 style={{ fontSize: '48px' }}>VR Games</h1>
+            <h1 style={{ fontSize: '48px' }}>VR</h1>
```

Comment:
```suggestion
```

Reply:
That was for when we were initially coding the page.